src/lib.rs

```diff
@@ -15,6 +15,11 @@ extern crate phf;
 #[macro_use]
 extern crate std;
 
+#[cfg(target_arch = "x86")]
+use core::arch::x86 as arch;
+#[cfg(target_arch = "x86_64")]
+use core::arch::x86_64 as arch;
```

Comment:
I think it would be more consistent to bind `arch` in each `bits*`.

Reply:
The issue is that even on x64, the `bits32` module gets compiled, but there is no `x86` submodule in `arch` then.

Reply:
Oh OK.